README.md

```diff
@@ -25,9 +25,26 @@
 - [Sending Payments](#sending-payments)
 - [Connector List](#connector-list)
 
+## Interledger Networks
+
+When moneyd was created, the Interledger livenet was a community-run network for research and development
+purposes. Now there is a well-established Interledger livenet based around wallet providers such as
+[Gatehub](https://gatehub.net), [XRPTipBot](https://xrptipbot.com), and [Stronghold](https://stronghold.co/).
+
+Today's Interledger livenet makes it easy for anyone to get a payment pointer to receive Interledger payments, and
+has handled billions of micropayments. However, the wallets which now make up the livenet have not yet exposed
+moneyd-like direct Interleger access.
+
+As a result, moneyd will only run against a community network of connectors, not the Interledger livenet.
+It's still a useful tool for experimenting with Interledger, but it is not currently suitable for any production
```

Comment:
```suggestion
Moneyd is still a useful tool for experimenting with Interledger, but it is not currently suitable for any production
```

Clarify, because `It` could refer to either "moneyd" or "the Interledger livenet" here.